d2l-input-time.html

```diff
@@ -0,0 +1,392 @@
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="d2l-input-shared-styles.html">
+<link rel="import" href="../d2l-dropdown/d2l-dropdown.html">
+<link rel="import" href="../d2l-dropdown/d2l-dropdown-content.html">
+<link rel="import" href="../iron-a11y-keys-behavior/iron-a11y-keys-behavior.html">
+<link rel="import" href="../iron-selector/iron-selector.html">
+<link rel="import" href="../d2l-typography/d2l-typography-shared-styles.html">
+<link rel="import" href="../d2l-intl-import/d2l-intl.html">
+
+<!--
+`d2l-input-time`
+Accessible, Localized Time Picker Input Element
+
+@demo demo/d2l-input-time.html
+-->
+
+<dom-module id="d2l-input-time">
+	<template strip-whitespace>
+		<style include="d2l-input-styles">
+			:host {
+				--d2l-input-line-height: 1;
+				--d2l-input-padding: 0.7rem 1rem;
+				--d2l-input-padding-focus: calc(0.7rem - 1px) calc(1rem - 1px);
+
+				display: inline-flex;
```

Comment:
Is this all pre-existing code.

Having `display: inline-flex;` on the `host` worries me a little.  If the consumer happens to put `display: inline-block;` on `d2l-input-time`, what affect would that have on the internal layout for the component?  For specifying internal layout, I think it's better to isolate internals from the public API to avoid risk of stomping on consumer, and vice versa.
```
host
   shadow-DOM
      div (where flex or whatever is defined)
         things that flex
```
Thoughts?

Reply:
Yup, all pre-existing code.  I'm not sure the reason for `inline-flex`, `inline-block` seems to do the same thing other than 10px of padding at the bottom disappears.  Feels weird making an issue for this, but I'm not sure if this breaks a usage in a form somewhere